tools/snippet-testing/snippet_tester.rb

```diff
@@ -61,6 +61,16 @@ def test_folder(snippet)
     end
   end
 
+  def add_hosts()
+    Dir.chdir(__dir__) do
+      `sudo -- sh -c "cat ../hosts >> /etc/hosts"`
```

Comment:
The problem when running only `sudo cat ../hosts >> /etc/hosts` is that `sudo` will scope only `cat` but the pipe to redirect output `>>` will be out of `sudo`, so basically `sudo` will prompt the user password to run just `cat`. Scoping everything into a single command shell make this work.